app/styles/app.css

```diff
@@ -461,3 +461,56 @@ pre .logger.log-log svg {
   @apply translate-x-[10px] translate-y-[10px];
   @apply dark:[color-scheme:dark];
 }
+
+/* Markdown Alerts */
+.markdown-alert {
+  @apply border-l-4 pl-4 py-1.5 my-2 dark:bg-gray-900 rounded-r-sm;
```

Comment:
The background on when in dark mode is not something Github does. I chose to add it in so it better stood out on the page since our page's dark background makes it all blend in like regular page content.